prime-router/metadata/fhir_mapping/hl7/resource/ServiceRequest.yml

```diff
@@ -133,11 +133,21 @@ authoredOn:
     dateTimeIn: ORC.9 | OBR.6
 
 occurrenceDateTime:
+  condition: $startTiming NULL && $endTiming NULL
   type: STRING
   valueOf: "GeneralUtils.dateTimeWithZoneId(dateTimeIn,ZONEID)"
   expressionType: JEXL
   vars:
-    dateTimeIn: ORC.15 | OBR.7 | OBR.27.4
```

Comment:
OBR.27 looks like it has a different mapping here now, but I should be able to cover that in my ticket, so this looks fine for now.